pkg/connector/user.go

```diff
@@ -21,12 +23,67 @@ type userResourceType struct {
 	client           *slack.Client
 	enterpriseID     string
 	enterpriseClient *enterprise.Client
+	ssoEnabled       bool
 }
 
 func (o *userResourceType) ResourceType(_ context.Context) *v2.ResourceType {
 	return o.resourceType
 }
 
+func (o *userResourceType) scimUserResource(ctx context.Context, user enterprise.UserResource) (*v2.Resource, error) {
+	// Get primary email
+	var primaryEmail string
+	for _, email := range user.Emails {
+		if email.Primary {
+			primaryEmail = email.Value
```

Comment:
we should add all the emails? Why just primary? Maybe we can add all emails and have a "primary_email" profile field? Having access to all the emails often helps us troubleshoot duplicate user problems in C1